app/views/parcels.py

```diff
@@ -0,0 +1,111 @@
+from flask import jsonify, request, Blueprint
+from app.model.parcel import Parcel
+from app import *
+
+ap = Blueprint('endpoint', __name__)
+PARCEL = Parcel()
+
+
+@ap.route("/")
+def welcome():
+    return jsonify({"message": "Welcome to the sendit api,v1"}),200
+
+
+# GET parcels
+@ap.route('/api/v1/parcels')
+def get_parcels():
+    '''
+    returns a list of all order requests
+    '''
+    all = PARCEL.get_all_parcel()
+    if all:
+        return jsonify({'count': len(all), 'orders': all}), 200
+    return jsonify({'msg': 'no parcel delivery orders posted yet', 'count': len(all)}), 404
+
+
+# GET parcels/id
+@ap.route('/api/v1/parcels/<int:id>')
+def get_a_parcel(id):
+    '''
+    return order request details for a specific order
+    '''
+    if not PARCEL.is_parcel_exist(id):
+        return jsonify({"msg": "parcel delivery request order not found"}), 404
+    return jsonify(PARCEL.get_one_parcel(id)), 200
+
+
+# POST /parcels
+@ap.route('/api/v1/parcels', methods=['POST'])
+def add_parcel():
+    '''
+    creates a new parcel order
+    '''
+    if not request.content_type == 'application/json':
+        return jsonify({"failed": 'Content-type must be application/json'}), 415
```

Comment:
Error code 415, is used for un supported 'media' type, That means it deals with files,images and others. use 400, instead which stands for a bad request the user is making

Reply:
Nice work